app/Services/Model/Imp/SummitOrderService.php

```diff
@@ -2331,7 +2331,8 @@ private function createTicketsForOrder
                         throw new EntityNotFoundException(sprintf("Promo code %s not found.", $promo_code));
                     }
                     Log::debug(sprintf("SummitOrderService::createTicketsForOrder applying promo code %s", $pc->getCode()));
-                    $pc->addUsage(1);
+                    $owner_email = !is_null($attendee) ? $attendee->getEmail() : $order->getOwnerEmail();
+                    $pc->addUsage(1, $owner_email);
```

Comment:
lets invert the signature of addUsage
addUsage($owner_email, $qty = 1) so we could invoke here
like
$pc->addUsage($owner_email)

Reply:
done